Cython/Utility/Exceptions.c

```diff
@@ -773,7 +773,7 @@ static int __Pyx_CLineForTraceback(PyThreadState *tstate, int c_line);/*proto*/
 //@requires: PyErrFetchRestore
 //@substitute: naming
 
-#ifndef CYTHON_CLINE_IN_TRACEBACK
+#if !(CYTHON_CLINE_IN_TRACEBACK == 0 || CYTHON_CLINE_IN_TRACEBACK == 1)
```

Comment:
The macro won't be undefined because it gets defined in `ModuleNode` (if it isn't already defined).

However it sounds like I should revert this bit of the change anyway so it doesn't matter